config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -208,14 +208,22 @@ private Optional<ConfigDocument> getLatestVersionConfigDoc(
 
   private Map<ConfigResourceContext, Optional<ConfigDocument>> getLatestVersionConfigDocs(
       Set<ConfigResourceContext> configResourceContexts) throws IOException {
+    if (configResourceContexts.isEmpty()) {
```

Comment:
Should we be treating an updateAll with empty as valid? or rejecting the request?

Reply:
Hmm🤔. We reject the request when we try to bulk upsert entities with empty request and also doc store `IN` operator does not take empty list as a valid request. Maybe here also we should reject the request? What do you think?

Reply:
I would think we should be rejecting - it seems unintentional.

Reply:
Sure. I've added this check with this PR - https://github.com/hypertrace/config-service/pull/152. Please review when possible.